readme.md

```diff
@@ -378,6 +378,19 @@ alfy.config.get('unicorn');
 //=> '🦄'
 ```
 
+#### userConfig
```

Comment:
Not sure if we have to document how this works? By creating a `config.json` file in the root of your package? Or link directly to https://github.com/SamVerschueren/alfred-config#workflow-configuration?

Reply:
I would add a short description of how it works, and then link to alfred-config for more details. I would also include the use-cases here. Not clear from the property what a Alfred workflow creator would use it for when they read the docs.

https://github.com/SamVerschueren/alfred-config#workflow-configuration should describe where to put the JSON file, etc.